dd-sdk-android/src/main/kotlin/com/datadog/android/v2/core/DatadogCore.kt

```diff
@@ -61,6 +62,9 @@ class DatadogCore(
     internal var webViewLogsFeature: SdkFeature<JsonObject, Configuration.Feature.Logs>? = null
     internal var webViewRumFeature: SdkFeature<Any, Configuration.Feature.RUM>? = null
 
+    // TODO RUMM-0000 handle context
```

Comment:
RUM-0000 ?

Reply:
yes, it is used like that in some place in the codebase, for the things which have task missing. Not sure where context will be handled, maybe in some existing task.